src/DynamoCoreWpf/Views/PackageManager/TermsOfUseView.xaml.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+using Dynamo.ViewModels;
+using Dynamo;
+
+namespace Dynamo.PackageManager.UI
+{
+    /// <summary>
+    /// Interaction logic for TermsOfUseView.xaml
+    /// </summary>
+    public partial class TermsOfUseView : Window
+    {
+        private DynamoViewModel dynamoViewModel;
+
+        public TermsOfUseView(DynamoViewModel dynamoViewModel)
```

Comment:
This dialog shouldn't have dependency on `DynamoViewModel`, please remove the `dynamoViewModel`. Instead of this, the dialog should have a Boolean property:

```csharp
bool TermsAccepted { get; private set; }
```

In the constructor of `TermsOfUseView`, this is set to `false`. The caller can then use this property to determine, after the window is closed, if the user has accepted the terms, then it is the caller's responsibility to update `PackageDownloadTouAccepted` instead.